liesel/distributions/mvn_degen.py

```diff
@@ -95,8 +103,37 @@ class MultivariateNormalDegenerate(tfd.Distribution):
       operation and can be avoided by specifying the corresponding arguments.
     * When you draw samples from the distribution via :meth:`.sample`, it is always
       necessary to compute the eigendecomposition of the distribution's precision
-      matrices once and cache it, because sampling requires both the eigenvalues
-      and eigenvectors.
+      matrices once and cache it, because sampling requires both the eigenvalues and
+      eigenvectors.
+
+    **Details on sampling**
+
+    If we have a singular precision matrix :math:`P`, we can view it as the
+    pseudo-inverse :math:`\\Sigma^+` of the variance-covariance matrix :math:`\\Sigma`.
+    We can obtain :math:`\\Sigma` by finding the singular value decomposition
+
+    .. math::
+        \\Sigma^+ = QA^+Q^T,
+
+    where :math:`Q` is the orthogonal matrix of eigenvectors and :math:`A^+` is the
+    diagonal matrix of singular values of :math:`\\Sigma^+ = P`. If the precision matrix
```

Comment:
P is square. Thus, I would refer to EVD instead of SVD. Also, above you refer to eigenvals and not SVs.